tests/integration/targets/etcd3/aliases

```diff
@@ -1,5 +1,8 @@
-destructive
 shippable/posix/group1
+destructive
+#needs/file/tests/utils/constraints.txt
+needs/target/setup_etcd3
+skip/aix
 skip/osx
 skip/freebsd
-disabled
+skip/python2.6  # lookups are controller only, and we no longer support Python 2.6 on the controller
```

Comment:
I don't think this line is needed.

Reply:
You're right: bad copy/paste  ;)
Howerver, etcd3 python module only supports 2.7, So I assume python 2.6 should kept skipped. By the way nowadays, who use ansible < 2.7 ???

Reply:
Regarding the documentation, Ansible officially supports 2.6 on nodes `You also need Python 2 (version 2.6 or later) or Python 3 (version 3.5 or later).`. For controllers 2.7 and higher.
It's just FYI, i don't insist on anything related:)

Reply:
Ok, this line probably is needed, but definitely *not* with this comment :) The only 2.6 integration target is centos/6, and currently the tests fail there.

Reply:
Yes, as expected, etcd3 python module fails to install on python < 2.7, probably because of grpcio dependency.
I did not pushed the tests (and I don't want to) but by the way, IMHO, etcd3 is a recent technology and I can't imagine a use-case that would require to etcd3 from obsolete servers.

Good to remind that RHEL/CentOS 6 will be retired by the end of this year (2020-11-30)

Reply:
From what I heard, there are still a lot of even older systems out there than RHEL/CentOS 6... But yeah, I guess these have enough other problems already ;)